### ISSUE TITLE
NTR: Cutaneous granulomatosis

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#10682]
Preferred term label:
Cutaneous granulomatosis

Synonyms
granulomatous skin disease, granulomatous inflammation of the skin

Definition (free text, please give PubMed ID)
Skin inflammatory reaction triggered by a wide variety of stimuli, including infections, foreign bodies, malignancy, metabolites, and chemicals. From a pathogenic point of view, they are divided into non-infectious and infectious granulomas. From a clinical point of view, it is useful to divide cutaneous granulomatosis into localized and more disseminated forms, although this distinction can be sometimes artificial.  **PMID: 29352388**

Parent term (use hpo.jax.org/app)
Granulomatosis HP:0002955, granuloma annulare MONDO:0006554, 

Sub-terms, requested
1. Non-infectious granulomatous skin diseases include granuloma annulare, necrobiosis lipoidica, rheumatic nodules, foreign body granulomas, cutaneous sarcoidosis, and interstitial granulomatous dermatitis. 
2. Infectious granulomas of the skin are caused by mycobacteria, in particular Mycobacterium tuberculosis or atypical mycobacteria; parasites, such as Leishmania; or fungi. 
3. Localized granulomatous lesions can be distinguished in three types: palisaded granulomas (granuloma annulare, necrobiosis lipoidica, and rheumatoid nodules), foreign body granulomas, and infectious granulomas, which are generally associated with localized infections. 
4. Disseminated cutaneous granulomas can be divided into infectious, in particular tuberculosis, and non-infectious forms, among which sarcoidosis and interstitial granulomatous dermatitis. 

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)
granulomatous dermatitis, MONDO:0006555

Your nano-attribution (ORCID)
0000-0002-7371-8158

**Comment by pnrobinson** [comment]
@GrieseAG 
Some of the above subterms seem to be disease entitities rather than phenotypic features.
I would suggest we make terms
Localized cutaneous granulomatosis
Disseminated cutaneous granulomatosis

For now I have this:
Adding new term: 
- ID: HP:6001214
- label: Cutaneous granulomatosis
- parents: [Granulomatosis (HP:0002955)](https://hpo.jax.org/app/browse/term/HP:0002955)
- pmid: PMID:29352388
- def: Skin inflammatory reaction triggered by a wide variety of stimuli, including infections, foreign bodies, malignancy, metabolites, and chemicals. From a pathogenic point of view, they are divided into non-infectious and infectious granulomas. From a clinical point of view, it is useful to divide cutaneous granulomatosis into localized and more disseminated forms, although this distinction can be sometimes artificial.
- comment: 
- orcid: orcid:0000-0002-7371-8158
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10682
- synonyms: 


Thoughts?

**Comment by pnrobinson** [comment]
@GrieseAG ping?

**Comment by GrieseAG** [comment]
Thank you, Peter, for the clarification. I agree—two terms are sufficient. You can upload the terms as suggested. 
Best, Christina

**Comment by pnrobinson** [comment]
OK, closing!
